frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -27,7 +27,6 @@ local MODE = {
     book_time_to_read = 6,
     chapter_time_to_read = 7,
     frontlight = 8,
-    mem_usage = 9,
```

Comment:
I also would leave memory info, it can be disabled by default, but available to be included in footer for users that want it (i.e. I want it :D)